bundles/org.openhab.binding.pushbullet/src/main/java/org/openhab/binding/pushbullet/internal/handler/PushbulletHandler.java

```diff
@@ -87,9 +87,35 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void initialize() {
         logger.debug("Start initializing!");
-        config = getConfigAs(PushbulletConfiguration.class);
+        PushbulletConfiguration config = getConfigAs(PushbulletConfiguration.class);
+
+        httpClient.setConfiguration(config);
+
+        if (config.getToken().isEmpty()) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Undefined token.");
+            return;
+        }
+
+        User user = httpClient.executeRequest(API_ENDPOINT_USERS_ME, User.class);
```

Comment:
`initialize()` should return fast. This call is blocking. It should be run in async. 

I think that from this line to the end of the block, could be extracted to a seperate private method that can be called from a `scheduler.execute` task